debug-scripts/get_all_logs.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+usage()
+{
+	echo "Usage: bash get_all_logs.sh <application_pod> <application_pod_namespace> <output_file>"
+	echo "Note: This script doesn't do any kind of input validation, and, also, doesn't modify the cluster"
+	exit 1
+}
+
+get_kubectl_cmds_output()
+{
+	app_pod=$1
+	app_ns=$2
+	operator_ns="openebs"
```

Comment:
openebs operator namespace can be different. an optional parameter can be provided to over-ride the default of openebs.

Reply:
to make it to give minimal options, is there a way to find the operator namespace by using some kubectl commands?